### PR TITLE
peer review

### PR DESCRIPTION
**Pull request by stefanotorresi**
- fix intro to reference the "it's just Angular" thing
- add some backticks notation to technical details
- add some emphasis to main concepts
- add some links
- some english tweaks

@simfin76 check this out!

### CODE REVIEW

**Comment by Jean85** [comment]
I've added some fixes!

File: content/blog/eng/angular1-component-a-path-to-angular2.md [hunk 1]
```diff
@@ -121,14 +123,14 @@ app.component('bar', {
 
 ```
 
-This would have created a two-way binding with the component’s parent. Whenever the parent would assign a new value, it would be propagated to the child. 
-And vice-versa, if the component assigns a new value it will be copied to the parent, this is two-way data bindings.
+This would have created a **two-way binding** with the component’s parent. Whenever the parent would assign a new value, it would be propagated to the child. 
+And vice-versa, if the component assigns a new value it will be copied to the parent; this is how a two-way data binding operates.
 
 This, while helpful, isn’t a very common scenario in my experience. That’s why Angular has finally introduced the new prospect of one-way bindings.
```
Comment (Jean85) [inline content/blog/eng/angular1-component-a-path-to-angular2.md]: Are we sure about this `prospect` word? I'm not sure about the meaning in this context

**Comment by stefanotorresi** [comment]
New guidelines for Angular naming recently came out:
http://angularjs.blogspot.it/2017/01/branding-guidelines-for-angular-and.html

We should comply.

**Comment by Jean85** [comment]
I've commited the latest fixed due to the naming convention. The remaining naming should remain in place, IMHO, due to version specific writing.

Apart from my comment in the review, LGTM :+1:

**Comment by Jean85** [comment]
Good!
Anyone else wants to suggest other changes? Or are we good to merge?

**Comment by Jean85** [comment]
Merged manually to avoid conflicts.

**Pull request closed without merging**